src/fdk_rdf_parser/rdf_utils/ns.py

```diff
@@ -43,3 +43,14 @@ def provURI(subString: str) -> URIRef:
 
 def dcatApiURI(subString: str) -> URIRef:
     return URIRef("http://dcat.no/dcatapi/" + subString)
+
+
+def rovURI(subString: str) -> URIRef:
+    return URIRef("http://www.w3.org/ns/regorg#" + subString)
```

Comment:
Optionally consider Python 3.6+ f-Strings
```suggestion
    return URIRef(f"http://www.w3.org/ns/regorg#{subString}")
```

Reply:
fa6f11a